examples/Atlas/+atlasParams/Base.m

```diff
@@ -3,11 +3,12 @@
   % are fixed at compile time for a particular type of plan (walking, manipulating,
   % standing, recovery, etc.) but which can be switched out for a different set when
   % the plan type changes. 
-  properties(GetAccess=public, SetAccess=protected)
+  properties
```

Comment:
There was no reason for the properties to be protected in the first place, and making them public made it easier to write unit tests that varied the properties without making a whole bunch of new param sets